packages/header/src/Header.tsx

```diff
@@ -27,208 +27,105 @@ const Icons = styled.div`
   }
 `
 
-const AppSelectorWrapper = styled.div`
-  position: absolute;
-  top: 60px;
-  left: 0;
-  min-width: 300px;
-  max-width: 300px;
-  background: #ffffff;
-  border: 1px solid gray;
-  display: flex;
-  flex-wrap: wrap;
-  flex-direction: row;
-  padding: 10px;
-  width: min-content;
-`
-
-const AppBox = styled.div`
-  border: 3px solid grey;
-  padding: 8px;
-  margin: 5px;
-  height: 80px;
-  width: 80px;
-  background: #b3dae0;
-  color: black;
-  display: flex;
-  align-items: center;
-  justify-content: center;
-  cursor: pointer;
-
-  &:hover {
-    background: #4f878d;
-  }
-`
 const ClickableIcon = styled.div`
   &:hover {
     color: gray;
     cursor: pointer;
   }
 `
 
-const UserInfoLabel = styled.b`
-  margin-left: 5px;
-`
-
-const Row = styled.div`
-  display: flex;
-  flex-direction: row;
-`
-
-const UnstyledList = styled.ul`
-  margin: 0;
-  padding: 0;
-  list-style-type: none;
-  display: inline-flex;
-`
-
 type THeaderPluginConfig = {
+  uiPluginsList: string[]
   hideUserInfo: boolean
   hideAbout: boolean
 }
 
-type TApplication = {
-  name: string
-  dataSources: string[]
-  label: string
-  description: string
-  roles: string[]
-  adminRole: string
-}
-
 export default (props: IUIPlugin): JSX.Element => {
   const { idReference, config: passedConfig } = props
-  const config: THeaderPluginConfig = passedConfig
-  const [entity, isLoading] = useDocument<TApplication>(idReference)
-  const { tokenData, token, logOut } = useContext(AuthContext)
+  const config = passedConfig as THeaderPluginConfig
+  const [entity, isApplicationLoading] = useDocument<TApplication>(idReference)
+
+  const path = 'dmss://DemoDataSource/DemoPackage/ExampleApplication'
+  const idRef = 'DemoDataSource/46a2c6ce-5bb4-41bc-923e-ede8dc95500e'
```

Comment:
Not sure how to deal with these two hard coded values. Perhaps let is be part of config instead?